modules/deploy/partials/cloud/tiers.adoc

```diff
@@ -19,6 +19,7 @@ When you create a BYOC cluster, you select your usage tier. Each tier provides t
 
 [NOTE]
 ====
+* On Azure, tiers 1-3 are currently supported. 
```

Comment:
```suggestion
* On Azure, tiers 1-3 are supported. 
```

Reply:
I'm curious why you used a table in the tabs here? Only one column suggests that each tab could simply be a list, no?